### PR TITLE
[#345] Removes Docker images and database on every deploy

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#345]

- [x] Modify `cjl` script to accept `-y` flags to prevent input prompt on the `clean` and `reset-db` commands.
- [x] Applies both modified `cjl` commands to continous deployment strategy

### CODE REVIEW

**Comment by lancelafontaine** [comment]
Since each service is running in Travis as it's own job in parallel, we may need some kind of homemade semaphore that make sure that this only happens once, for the last job to be completed. May require a future PR once this one is merged.

**Review by dodoels — APPROVED** [review]

**Review by Vynny — APPROVED** [review]

File: cjl [hunk 1]
```diff
@@ -7,27 +7,46 @@ fi
...
     echo "  >>> Killing all docker containers <<<" && echo "" && docker ps -qa | xargs docker kill && echo ""
     echo "  >>> Removing all stopped containers <<<" && echo "" && docker ps -qa | xargs docker rm -v && echo ""
     echo "  >>> Removing all images <<<" && echo "" && docker images -qa | xargs docker rmi && echo ""
+  }
 
+  if [ "$2" != "-y" ]; then
+    echo "WARNING: This command is destructive. It removes all containers and images installed on your local machine."
+    read -r -p "Are you SURE you want to continue? [y/N]" clean_confirmation
+    if [[ $clean_confirmation =~ ^(y|Y)$ ]]; then
+      clean
```
Comment (arekmano) [inline cjl:20]: why call it 2x?

**Review by arekmano — CHANGES_REQUESTED** [review]

File: cjl [hunk 2]
```diff
@@ -7,27 +7,46 @@ fi
...
+      clean
+    else
+      echo "Clean aborted."
+    fi
   else
-    echo "Clean aborted."
+      clean
```
Comment (arekmano) [inline cjl:26]: same as above

File: cjl (hunk 1 above)
Comment (lancelafontaine) [reply cjl:20]: Probably not needed, but I've sometimes some situations where more images get removed if you do it a second time, I think in cases when there are "nested images" (?). Regardless, no harm in running it twice, and our script should try to enforce the state we want.

**Review by arekmano — APPROVED** [review]
ok

**Pull request merged**